mypy/suggestions.py

```diff
@@ -224,7 +241,9 @@ def get_args(self, is_method: bool,
                 types.append(generate_type_combinations(all_arg_types))
             else:
                 # If we don't have anything, we'll try Any and object
-                types.append([AnyType(TypeOfAny.explicit), self.builtin_type('builtins.object')])
+                # (Actually object usually is bad for downstream consumers...)
+                # types.append([AnyType(TypeOfAny.explicit), self.builtin_type('builtins.object')])
```

Comment:
Maybe commit to not doing object rather than commenting it out and making the comment contradict itself?